File: Amazaing-Management-frontend/src/app/models/employee-dto.ts
export class EmployeeDTO {

    constructor(
        private _employeeName: string,
        private _photo: string,
        private _productivity: number,
        private _clumsiness: number,
        private _companyId: number,
        private _playerId: number
    ) { }

    public get playerId(): number {
        return this._playerId;
    }
    public set playerId(value: number) {
        this._playerId = value;
    }
    public get companyId(): number {
        return this._companyId;
    }
    public set companyId(value: number) {
        this._companyId = value;
    }
    public get clumsiness(): number {
        return this._clumsiness;
    }
    public set clumsiness(value: number) {
        this._clumsiness = value;
    }
    public get productivity(): number {
        return this._productivity;
    }
    public set productivity(value: number) {
        this._productivity = value;
    }
    public get photo(): string {
        return this._photo;
    }
    public set photo(value: string) {
        this._photo = value;
    }
    public get employeeName(): string {
        return this._employeeName;
    }
    public set employeeName(value: string) {
        this._employeeName = value;
    }

}

File: backend/company-service/src/main/java/com/ironhack/companyservice/controller/dto/EmployeeDTO.java
package com.ironhack.companyservice.controller.dto;

import java.math.BigDecimal;

public class EmployeeDTO {
	private Long employeeId;
	private String employeeName;
	private String photo;
	private BigDecimal productivity;
	private BigDecimal clumsiness;
	private Long companyId;
	private Long playerId;

	public EmployeeDTO() {
	}

	public EmployeeDTO(String employeeName, String photo, BigDecimal productivity, BigDecimal clumsiness, Long companyId, Long playerId) {
		this.employeeName = employeeName;
		this.photo = photo;
		this.productivity = productivity;
		this.clumsiness = clumsiness;
		this.companyId = companyId;
		this.playerId = playerId;
	}

	public Long getEmployeeId() {
		return employeeId;
	}

	public void setEmployeeId(Long employeeId) {
		this.employeeId = employeeId;
	}

	public String getEmployeeName() {
		return employeeName;
	}

	public void setEmployeeName(String employeeName) {
		this.employeeName = employeeName;
	}

	public String getPhoto() {
		return photo;
	}

	public void setPhoto(String photo) {
		this.photo = photo;
	}

	public BigDecimal getProductivity() {
		return productivity;
	}

	public void setProductivity(BigDecimal productivity) {
		this.productivity = productivity;
	}

	public BigDecimal getClumsiness() {
		return clumsiness;
	}

	public void setClumsiness(BigDecimal clumsiness) {
		this.clumsiness = clumsiness;
	}

	public Long getCompanyId() {
		return companyId;
	}

	public void setCompanyId(Long companyId) {
		this.companyId = companyId;
	}

	public Long getPlayerId() {
		return playerId;
	}

	public void setPlayerId(Long playerId) {
		this.playerId = playerId;
	}

}
